src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -101,6 +100,18 @@ export default function EnvironmentEditor({
             endIndex: result.destination.index,
         });
         setScenes(items);
+
+        const newSceneIds = items.map((scene) => scene.id);
+        const response = await editScenario({
+            id: environment.id,
+            name: environment.name,
+            friendly_name: environment.friendly_name,
+            description: environment.description,
+            scene_ids: newSceneIds,
+            is_published: environment.is_published,
+            is_previewable: environment.is_previewable,
+        });
```

Comment:
Not a fan of this, would have liked to do object destructuring like so:
```javascript
const { oldSceneIds, ...rest } = environment
const response = await editScenario({
    scene_ids: newSceneIds,
    ...rest,
})
```
Much cleaner but then I get warnings of unused variables :(

Reply:
I'm assuming the error was on `oldSceneIds` not being used?

If it's an eslint error, which I suspect it is, you can add the following line above it:
`// eslint-disable-next-line no-unused-vars`

And there should be no more warnings for that.

Reply:
Ah it's ok, not too big of a deal. Let's just merge it in. If we feel like we need a cleaner way, then we can change it then. I'm fine with the way I implemented it for now.